enterprise/frontend/src/metabase-enterprise/llm_autodescription/LLMSuggestQuestionInfo.tsx

```diff
@@ -45,7 +45,7 @@ export const LLMSuggestQuestionInfo = ({
     : t`Description generated. Click to auto-fill.`;
 
   const className = loading ? "llm-pulse-icon" : undefined;
-  const iconColor = loading ? color("text-medium") : color("brand");
+  const iconColor = loading ? color("text-medium") : "var(--mb-color-brand)";
```

Comment:
Should we convert `text-medium` in this PR as well, or in another PR? I saw #43512 changes `bg-medium`, but not sure which PR you want to address `text-medium`.

Reply:
Nice!